src/vt-tv/api/object_work.h

```diff
@@ -109,6 +110,60 @@ struct ObjectWork {
    */
   auto const& getUserDefined() const { return user_defined_; }
 
+  /**
+   * \brief set communications for this object
+   *
+   * \return void
+   */
+  void setCommunications(ObjectCommunicator c) {
```

Comment:
We could probably std::move this or at least take by reference to avoid necessary copies.

Reply:
I changed this to pass by reference for now, as this function isn't used yet.